src/main/webapp/app/shared/monaco-editor/model/actions/grading-criteria/monaco-grading-description.action.ts

```diff
@@ -0,0 +1,20 @@
+import * as monaco from 'monaco-editor';
+import { MonacoEditorDomainAction } from 'app/shared/monaco-editor/model/actions/monaco-editor-domain-action.model';
+
+export class MonacoGradingDescriptionAction extends MonacoEditorDomainAction {
+    static readonly ID = 'monaco-grading-description.action';
+    static readonly IDENTIFIER = '[description]';
+    static readonly TEXT = 'Add grading instruction here (only visible for tutors)';
+
+    constructor() {
+        super(MonacoGradingDescriptionAction.ID, 'artemisApp.assessmentInstructions.instructions.editor.addDescription', undefined, undefined, true);
```

Comment:
why do we pass two undefined values here?

Reply:
The action has no icon and no keybindings -> `undefined`

Reply:
alright 👍